MIGRATING.md

```diff
@@ -1,5 +1,8 @@
 ## Migration Guides
 
+### Migrating from versions < 17.0.0
```

Comment:
I'm kind of inclined not to have another major version bump so soon...Maybe `16.1.0`? This should only be a breaking change for Connect users right? Thoughts @yuki-stripe @davidme-stripe

Reply:
I'm not familiar with how versioning has worked here in the past, but moving from the 2015 API sounds like an appropriate reason to increment the major version number.

Reply:
Definitely wary of fatiguing our users...there are the (minor) enum changes that Source users should update with, too.  Outside of that, it *should* be unnoticed. 

Either way, I'd like to ship this only after some confidence that 16.* won't have more bugfixes.

Reply:
👍 Let's hold this patch until 16.0.* has settled and then go from there